lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1912,8 +1919,8 @@ StringAttr MemOp::getPortName(size_t resultNo) {
   return getPortNames()[resultNo].cast<StringAttr>();
 }
 
-FIRRTLType MemOp::getPortType(size_t resultNo) {
-  return getResults()[resultNo].getType().cast<FIRRTLType>();
+FIRRTLBaseType MemOp::getPortType(size_t resultNo) {
```

Comment:
Note: MemOp might need a `RefType` port for MemTaps, depending on how we implement them.

Reply:
Leaving this alone for now-- until it's supported might as well keep it unsupported.  Will keep in mind though, and LMK if this seems worth addressing now!

Reply:
Yes, lets leave it out for now.